src/utils/flow.js

```diff
@@ -46,6 +46,21 @@ export function convertFlowObjectTypeAnnotation(
       };
     }
 
+    if (value.type === "ArrayTypeAnnotation" && value.elementType) {
+      const children = value.elementType.id && flowTypes[value.elementType.id.name]
+        ? flowTypes[value.elementType.id.name]
+        : value.elementType;
+
+      return {
+        ...obj,
+        [key]: {
+          type: "array",
+          nullable: property.optional,
+          children: children.properties ? convertFlowObjectTypeAnnotation(children, flowTypes) : null
```

Comment:
I'm good with it as a starting point as well.

I'm starting to think it might make sense to have some kind of `FlowTypeAnnotation` resolver that can handle `GenericTypeAnnotations`, and maybe even offer something similar to `ofType` that we can use for lists (that would use the resolver as well). Also thinking that `children` might need to be a `FlowType`, but that should be easier to address down the line once this resolving stuff is finished.